docs/cloud-tutorials/best-practices.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 8
+title: Best practices
+description: Tips and tricks for succesfully using DBOS Cloud
+---
+
+This section dives into specific behaviors and corner cases you might encounter when using DBOS Cloud.
+
+## Managing applications
+
+### Registration
+* An application must be registered before being deployed. DBOS Cloud CLI will handle registration by default when you deploy an application for the first time.
+* Only one request at a time can manipulate an application and you will get an `application is busy` error if you attempt multiple concurrent modifications.
+* Application names are unique within an [organization](#organizations-and-users-management)
+
+### Deployment
+* Interrupting a deployment request - for example, pressing control + C - will *not* cancel the deploy action on DBOS Cloud
+* DBOS Cloud will prune all development dependencies. Make sure to explicitly install any runtime dependency you need with `npm install`
+* Your application must not use more than 1GB of disk space
```

Comment:
Same suggestion as above, rephrase it to something like: "Each app is allocated with 1GB of disk space." so it sounds more positive.

Also mention the RAM and CPU limits, so people will know what to expect.